ui/ui-components/pages/index.tsx

```diff
@@ -1,16 +1,29 @@
 import { useRouter } from "next/router";
+import { useApi } from "../hooks/useApi";
 import Head from "next/head";
 import { Row, Col, Image, Button } from "react-bootstrap";
 
 export default function Page(props) {
   const router = useRouter();
   const { navigationMode, navigation } = props;
+  const currentStep = props.setupSteps.find(
+    (step) => !step.complete && !step.skipped
+  );
 
   let CTALink = "/session/sign-in";
   let CTAMessage = "Sign In";
   let CTATarget: string;
 
-  if (navigationMode === "authenticated") {
+  if (navigationMode === "config" && currentStep) {
+    CTAMessage = "Set Up Grouparoo";
+    CTALink = "/setup";
+  } else if (navigationMode === "config" && !currentStep) {
+    CTAMessage = "Configure Your Profiles";
+    CTALink = "/profile";
```

Comment:
I believe this should be `/profiles`
```suggestion
    CTALink = "/profiles";
```